.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: Report an issue or unexpected behavior that occurs within the application
+title: "[Bug]: "
+labels: "bug"
+body:
+  - type: checkboxes
+    attributes:
+      label: Guidelines
+      description: Please ensure you've completed all of the following.
+      options:
+        - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one i want to file, without success.
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: |
+          1. Go to '...'
+          2. Click on '....'
+          3. Scroll down to '....'
+          4. See error
+    validations:
+      required: true   
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: FreeTube Version
+      description: |
+        If using releases, enter the version.
+        If using nightly builds, enter commit hash or build number, you can find it via about page in the FreeTube application.
+      placeholder: v0.14.0, 8c4278
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Operating System Version
+      description: What operating system version are you using? On Windows, click Start button > Settings > System > About. On macOS, click the Apple Menu > About This Mac. On Linux, use lsb_release or uname -a.
+      placeholder: "e.g. Windows 10 version 1909, macOS Catalina 10.15.7, or Ubuntu 20.04"
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: 'Installation Method (If applicable)'
+      options:
+        - .AppImage
+        - .deb
+        - .dmg
+        - .exe
+        - Flathub
+        - Portable
+        - .rpm
+        - .zip
+        - other
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: 'Last Known Working FreeTube Version (If Any)'
+      description: What is the last version of FreeTube this worked in, if applicable?
+      placeholder: v0.14.0
+  - type: dropdown
+    attributes: 
+      label: Primary API used
+      description: What is the primary API being used?
+      multiple: false
+      options:
+        - Local API
+        - Invidious API
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Additional Information
```

Comment:
I would change this to: Screenshots / Additional Information

Reply:
Dont think that is a good idea

Check out line 81

https://github.com/FreeTubeApp/FreeTube/commit/aaa336bf48fc6bf20f1172d5f3d6a78aa3cb9bfc